packages/@ember/-internals/views/lib/system/event_dispatcher.js

```diff
@@ -335,7 +336,9 @@ export default EmberObject.extend({
           while (
             target &&
             target.nodeType === 1 &&
-            (!related || (related !== target && !target.contains(related)))
+            (!related ||
+              (related !== target &&
+                !(target.contains ? target.contains(related) : contains(target, related))))
```

Comment:
I'd _vaguely_ prefer to always use `contains(target, related)` and tweak the contains util to deal with the ternary, what do you think?

Reply:
Makes sense!

Reply:
Done!